mod/ckeditor/views/default/elgg/ckeditor/config.js

```diff
@@ -8,8 +8,8 @@ define(function(require) {
 		removeButtons: 'Subscript,Superscript', // To have Underline back
 		allowedContent: true,
 		baseHref: elgg.get_site_url(),
-		removePlugins: 'liststyle,contextmenu,tabletools',
-		extraPlugins: 'blockimagepaste,autogrow',
```

Comment:
I'm OK with this, but @Elgg/core should probably determine if the functionality these plugins were providing is still needed.